src/tools/kata-ctl/src/check.rs

```diff
@@ -144,6 +180,41 @@ pub fn check_version() -> Result<()> {
     Ok(())
 }
 
+pub fn check_all_releases() -> Result<()> {
+    let releases: Vec<Release> = get_kata_all_releases_by_url().map_err(handle_reqwest_error)?;
+
+    for release in releases {
+        if !release.prerelease {
+            println!(
+                "Official:   Release {:15}; created {} ; {}",
```

Comment:
nit: The white space number is different from `PreRelease: Release {:15}; created {} ; {}`, could you make it consistent?